### PR TITLE
[Feature #861][ALL] Remove useless imports

### LINKED ISSUE #861: [Feature][ALL] Remove useless imports
### Search before asking

- [X] I had searched in the [feature](https://github.com/InterestingLab/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Remove useless imports

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wuchunfu**
## Purpose of this pull request

close #861
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] Change does not need document change, or I will submit document change to https://github.com/InterestingLab/seatunnel-docs later

### CODE REVIEW

**Comment by leo65535** [comment]
hi @wuchunfu, can we add some scala checkstyle rule to check it?

**Comment by CalvinKirs** [comment]
> hi @wuchunfu, can we add some scala checkstyle rule to check it?

We have a test, but I don’t know why it didn’t take effect this time. Let me try again later.

**Comment by CalvinKirs** [comment]
> > hi @wuchunfu, can we add some scala checkstyle rule to check it?
> 
> We have a test, but I don’t know why it didn’t take effect this time. Let me try again later.

I saw it, he only checked java, I will modify it later

**Comment by CalvinKirs** [comment]
> > > hi @wuchunfu, can we add some scala checkstyle rule to check it?
> > 
> > 
> > We have a test, but I don’t know why it didn’t take effect this time. Let me try again later.
> 
> I saw it, he only checked java, I will modify it later

@wuchunfu Or can you add the changes here in your pr?

**Comment by wuchunfu** [comment]
> > > > hi @wuchunfu, can we add some scala checkstyle rule to check it?
> > > 
> > > 
> > > We have a test, but I don’t know why it didn’t take effect this time. Let me try again later.
> > 
> > 
> > I saw it, he only checked java, I will modify it later
> 
> @wuchunfu Or can you add the changes here in your pr?

Okay, I will add a `scalastyle.xml` to check it later.

**Comment by wuchunfu** [comment]
> > > > > hi @wuchunfu, can we add some scala checkstyle rule to check it?
> > > > 
> > > > 
> > > > We have a test, but I don’t know why it didn’t take effect this time. Let me try again later.
> > > 
> > > 
> > > I saw it, he only checked java, I will modify it later
> > 
> > 
> > @wuchunfu Or can you add the changes here in your pr?
> 
> Okay, I will add a `scalastyle.xml` to check it later.

I think I should submit another pr because there may be more changes after adding `scalastyle.xml` .

**Comment by CalvinKirs** [comment]
> > > > > > hi @wuchunfu, can we add some scala checkstyle rule to check it?
> > > > > 
> > > > > 
> > > > > We have a test, but I don’t know why it didn’t take effect this time. Let me try again later.
> > > > 
> > > > 
> > > > I saw it, he only checked java, I will modify it later
> > > 
> > > 
> > > @wuchunfu Or can you add the changes here in your pr?
> > 
> > 
> > Okay, I will add a `scalastyle.xml` to check it later.
> 
> I think I should submit another pr because there may be more changes after adding `scalastyle.xml` .

well, It's better

**Comment by wuchunfu** [comment]
cc @CalvinKirs

File: seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala [hunk 1]
```diff
@@ -114,8 +111,8 @@ object SparkBatchExecution {
   }
 
   private[seatunnel] def registerTransformTempView(
-      plugin: BaseSparkTransform,
-      ds: Dataset[Row]): Unit = {
+                                                    plugin: BaseSparkTransform,
+                                                    ds: Dataset[Row]): Unit = {
```
Comment (leo65535) [inline seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala:115]: the tab size seems a litte long.

Comment (wuchunfu) [reply seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala:115]: Ok i will fix it, thx

File: seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala [hunk 2]
```diff
@@ -80,9 +74,7 @@ object SparkBatchExecution {
     ds.createOrReplaceTempView(tableName)
   }
 
-  private[seatunnel] def registerInputTempView(
```
Comment (leo65535) [inline seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala:83]: Just a nit: I think we can keep it as it is, when the function has many args, the first form is better.
I am not sure if others prefer to the second form.
```
private[seatunnel] def registerInputTempView(
      source: BaseSparkSource[Dataset[Row]],
      environment: SparkEnvironment): Unit = {
```
```
private[seatunnel] def registerInputTempView(source: BaseSparkSource[Dataset[Row]], environment: SparkEnvironment): Unit = {
```

Comment (wuchunfu) [reply seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala:83]: Do I want to restore it? I want to hear your suggestions.

Comment (wuchunfu) [reply seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/batch/SparkBatchExecution.scala:83]: I also feel that the first one will be better.

**Review by CalvinKirs — APPROVED** [review]
LGTM
